readme.md

```diff
@@ -35,6 +35,7 @@ JAX brings automatic differentiation and the XLA compiler together through a num
 - [Chex](https://github.com/deepmind/chex) - utilities to write and test reliable JAX code.
 - [Optax](https://github.com/deepmind/optax) - a gradient processing and optimization library.
 - [JAX, M.D.](https://github.com/google/jax-md) - accelerated, differential molecular dynamics.
+- [jax-unirep](https://github.com/ElArkk/jax-unirep) - JAX implementation of the UniRep protein RNN.
```

Comment:
Perhaps we can change the description so it is more evident that it's a library and not just a re-implementation. I was thinking something like " - library implementing the [UniRep](https://www.nature.com/articles/s41592-019-0598-1) model for protein machine learning applications."

Reply:
I like that, @n2cholas! Thanks for the suggestion. Coming up!